### DISCUSSION TITLE
[探店]我以后不会再做这门生意了。

### DISCUSSION THREAD

**Discussion by nninng** [share121/inter-knot#4507] [其他]
今天上线发现我发的两期探店视频都被删了，既然有人不让我做那我就不做了，探店就此终结。

**Comment by KELERSAMA** [comment]
ewq 别啊 小n你的探店在绳网已经算高质量了 没了你我上哪看探店去啊😭

**Reply by nninng** [reply]
感谢您的支持，我会回归到探店事业当中的。

**Comment by share121** [comment]
很抱歉对您产生的影响。
此次事故的原因很有可能是脚本出现问题。在自动处理举报时，误删了很多帖子。
这次事故导致绳网上有将近 1000 个帖子被删除。
我的更新日志和举报处也被删除。

**Comment by Dave-12138** [comment]
我的表情包发布帖也无了（<img src="https://bbs-static.miyoushe.com/static/2024/09/04/ce696e4c9307a36be82d8410dcb96cc3_5924553387554391103.png" alt="卡齐娜-紧张" width="75" >

**Comment by nninng** [comment]
如果是针对网站下手的话，我会继续做探店的。
